packages/manager/apps/hub/src/index.html

```diff
@@ -45,7 +45,7 @@
                     event: 'sidebar:loaded',
                 },
             }"
-            data-sidebar-expand="true"
+            data-sidebar-expand="isDashboardState"
```

Comment:
```suggestion
            data-sidebar-expand="shouldExpandSidebar"
```
we might want in future cases to expand the sidebar